frontend/src/pages/Player/Toolbar/DevToolsWindowV2/EmptyDevToolsCallout/EmptyDevToolsCallout.tsx

```diff
@@ -34,6 +39,9 @@ export const EmptyDevToolsCallout = ({
 }: Props) => {
 	return (
 		<Box
+			style={{
+				maxWidth: '340px',
+			}}
```

Comment:
the callout is not centered

![Screenshot from 2023-09-05 10-30-39](https://github.com/highlight/highlight/assets/1351531/75a7c4c3-1130-4553-a781-9713ba576ea8)

![Screenshot from 2023-09-05 10-29-22](https://github.com/highlight/highlight/assets/1351531/b94732b2-7738-48bf-8089-52f82c05bac2)